demo/src/index.js

```diff
@@ -9,6 +9,7 @@ const config = {
   annotation: {
     adapter: (canvasId) => new LocalStorageAdapter(`localStorage://?canvasId=${canvasId}`),
     // adapter: (canvasId) => new AnnototAdapter(canvasId, endpointUrl),
+    exportLocalStorageAnnotations: false,
```

Comment:
```suggestion
    exportLocalStorageAnnotations: false, // display annotation JSON export button
```
This feature looks great. Just wanted to add a comment explaining this setting. I'm assuming this feature is mostly for debugging convenience and not meant for most users.

Reply:
Very good point, @camillevilla; thanks @mejackreed for the change.
We do actually have a request for that feature that goes a bit beyond debugging, but it's still not meant for permanent use or in a production environment, so yes, it's not meant for most users.